frontend/src/Services/api-service.js

```diff
@@ -27,6 +27,9 @@ export const getCategories = token =>
 export const getUser = (token, userName) =>
   axios.get(`/api/kirschbaum/user/${userName}`, headers(token))
 
+export const getAllUsers = token =>
+  axios.get(`/api/kirschbaum/user`, headers(token))
```

Comment:
```suggestion
  axios.get(`/api/kirschbaum/user`, headers(token)).then(response => response.data)
```

In most cases you are only interested in the data of a successful response. unpacking it in the service cleans up the calling code

Reply:
Changed it for all necessary requests, good point 👍

Reply:
Thanks for the effort 🙇